docs/root/intro/arch_overview/http/http_connection_management.rst

```diff
@@ -151,37 +151,56 @@ previously attempted priorities.
 Internal redirects
 --------------------------
 
-Envoy supports handling 302 redirects internally, that is capturing a 302 redirect response,
-synthesizing a new request, sending it to the upstream specified by the new route match, and
-returning the redirected response as the response to the original request.
+Envoy supports handling 3xx redirects internally, that is capturing a configurable 3xx redirect
```

Comment:
Do the API docs note that this is bounded to 3xx?  I don't see it documented in redirect_response_codes

Reply:
Added comment in the API proto as well.